build.sh

```diff
@@ -114,7 +114,13 @@ function do_init
 function prepare_build_dir
 {
   TYPE=$1
-  mkdir -p ${TOPDIR}/build_${TYPE} && cd ${TOPDIR}/build_${TYPE}
+  mkdir -p ${TOPDIR}/build_${TYPE}
+  if [ -e build ]; then
+      rm build
```

Comment:
It's better to use `rm -f` instead of `rm`.
Or remove the test branch and use `ln -sf` instead of `ln -s`.
I tested on my machine that if there already exits soft link, `ln -sf` works.

Reply:
> It's better to use `rm -f` instead of `rm`. Or remove the test branch and use `ln -sf` instead of `ln -s`. I tested on my machine that if there already exits soft link, `ln -sf` works.

I tested on my machine that when `build->build_debug`, if we execute `ln -sf build build_release`, the build still links to build_debug. So I will use `rm -f`.

Reply:
> It's better to use `rm -f` instead of `rm`. Or remove the test branch and use `ln -sf` instead of `ln -s`. I tested on my machine that if there already exits soft link, `ln -sf` works.

I tested on my machine that when `build->build_release`, if we execute `ln -sf build_debug build`, the `build` still links to build_release. So I will use `rm -f`